packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_panel.dart

```diff
@@ -115,22 +117,29 @@ class _PropertyEditorConnectedPanelState
             Scrollbar(
               controller: scrollController,
               thumbVisibility: true,
-              child: SingleChildScrollView(
-                controller: scrollController,
-                child: Padding(
-                  padding: const EdgeInsets.fromLTRB(
-                    denseSpacing,
-                    defaultSpacing,
-                    defaultSpacing, // Additional right padding for scroll bar.
-                    defaultSpacing,
+              child: Column(
+                children: [
+                  Expanded(
+                    child: SingleChildScrollView(
+                      controller: scrollController,
+                      child: Padding(
+                        padding: const EdgeInsets.fromLTRB(
+                          denseSpacing,
+                          defaultSpacing,
+                          defaultSpacing, // Additional right padding for scroll bar.
+                          defaultSpacing,
+                        ),
+                        child: Column(
+                          crossAxisAlignment: CrossAxisAlignment.start,
+                          children: [
+                            PropertyEditorView(controller: widget.controller),
+                          ],
+                        ),
+                      ),
+                    ),
                   ),
-                  child: Column(
-                    crossAxisAlignment: CrossAxisAlignment.start,
-                    children: [
-                      PropertyEditorView(controller: widget.controller),
-                    ],
-                  ),
-                ),
+                  const _PropertyEditorFooter(),
```

Comment:
can we use the existing `StatusLine` widget or at least its sub components?

Reply:
It looks like the `StatusLine` and its components expect there to be a `Screen`, which doesn't exist for the Property Editor or other panels - I think there would be some refactoring necessary to make that work but could do as a follow-up!

Reply:
sg to clean up later